db/migrate/20191028115549_create_application_qualification.rb

```diff
@@ -0,0 +1,15 @@
+class CreateApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_qualifications do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :qualification_type, null: false
+      t.string :subject, null: false
+      t.string :grade, null: false
+      t.string :award_year, null: false
+      t.string :institution_details
```

Comment:
On the form it's possible to enter the institution name and country separately — does it make sense for this to be a single field?

Reply:
We've split this into `institution_name` and `institution_country`.